src/app/components/Header/OpenWindowButton.vue

```diff
@@ -4,13 +4,8 @@
     style="float: right;"
     @click="$emit('input', !value)"
     :aria-label="`Click to make the chat ${value ? 'fullscreen' : 'normal size'}`"
+    :class="{ tabbing: $vxm.chat.tabbing }"
```

Comment:
As a general rule, I prefer to limit the amount of "globals", since it's not clear where they come from. I'd suggest a mixin, but at that point it's balancing whether that's beneficial for a whopping two lines.